mtags/src/main/scala-2/scala/meta/internal/pc/SemanticdbTextDocumentProvider.scala

```diff
@@ -46,7 +46,7 @@ class SemanticdbTextDocumentProvider(
 
     val explicitDialect = if (filePath.isSbt) {
       Some(dialects.Sbt1)
-    } else if (filePath.isScalaScript) {
+    } else if (filePath.isMill || filePath.isScalaScript) {
```

Comment:
Even though we try to consider `.mill` not scala scripts, they are actually Scala 2.13 files with top level terms, so I believe this should stay.